src/components/splitButton/splitButton.scss

```diff
@@ -0,0 +1,308 @@
+@use '../../scss/global';
+@use '../../scss/mixins/typography';
+
+:host {
+  display: inline-block;
+}
+
+/* ----------------- Split btn -------------------- */
+.split-btn-wrapper {
```

Comment:
Wrapper needs `width: 100%`, so devs can specify the size of the button on the host.

Reply:
Done